app/services/data/get-closed-claim-status.js

```diff
@@ -20,14 +20,6 @@ module.exports = function (claimId) {
     .andWhere('ClaimId', claimId)
     .orderBy('ClaimEventId', 'desc')
     .then(function (event) {
-      if (event) {
-        if (event !== null && event !== undefined) {
-          return (closedClaimStatusMap[event.Event])
-        } else {
-          return 'Closed'
-        }
-      } else {
-        return 'Closed'
-      }
+      return (event && event !== null && event !== undefined) ? closedClaimStatusMap[event.Event] : 'Closed'
```

Comment:
Will delete this, just left the file for now so you can see the simplification of the original call.

Reply:
Nice!